### PR TITLE
modification of method decompose and decomposition method tests

### PR DESCRIPTION
**Pull request by vcristiani**
I made a modification to the decompose method, to make the implementation of the scikit-learn wrappers easier.

### CODE REVIEW

**Review by nvillagra — COMMENTED** [review]
Everything looks fine

File: galaxychop/models.py [hunk 1]
```diff
@@ -41,6 +42,18 @@ class Columns(enum.Enum):
 class GCDecomposeMixin:
     """Galaxy chop decompose mixin class."""
 
+    def get_clean_mask(self, X):
+        """Clean the Nan value of the circular parameter array."""
+        eps = X[:, Columns.circular_parameter.value]
+        (clean,) = np.where(~np.isnan(eps))
```
Comment (ornemarioni) [inline galaxychop/models.py:48]: check if  (clean,) = np.where(~np.isnan(eps)) is not the same of clean =~np.isnan(eps)

**Review by ornemarioni — APPROVED** [review]
all looks fine!

**Review by antotaverna — APPROVED** [review]
Everything looks good

**Pull request merged**